lib/etcconf/hosts.go

```diff
@@ -145,12 +144,6 @@ func (h *hosts) Save() error {
 		return err
 	}
 
-	// make sure the file is readable
```

Comment:
I think you should leave this here, since this code should run independent of bind-mount. You can keep the chmod after bind-mount elsewhere.